backstop/assets/theme.js

```diff
@@ -1,88 +1,126 @@
-const THEMEAVAILABLE = ['iui-theme-light', 'iui-theme-dark', 'iui-theme-light-hc', 'iui-theme-dark-hc'];
-
 class ThemeButton extends HTMLElement {
   constructor() {
     super();
-    const html = `
-      <button title="Toggle theme" id="theme-button">
-        <svg viewBox="0 0 16 16" aria-hidden="true">
-          <path d="m8 0a8 8 0 1 0 8 8 8 8 0 0 0 -8-8zm0 12v3a7 7 0 0 1 0-14v3a4 4 0 0 1 0 8zm0 0a4 4 0 0 1 0-8z"/>
-        </svg>
-        <span class="toggle-theme-label">Toggle theme</span>
-      </button>`;
-    const style = `
-      :host {
+    const html = /* html */ `
+      <div class="settings-root">
+        <button aria-label="Settings">
+          <svg width="1rem" height="1rem" fill="currentColor" aria-hidden="true" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
+            <path d="m16 9.42256v-2.85566l-2.20352-.44435a6.05356 6.05356 0 0 0 -.37645-.903l1.2427-1.87048-2.01923-2.01931-1.86669 1.24016a6.047 6.047 0 0 0 -.91294-.38153l-.44131-2.18839h-2.85566l-.44131 2.18839a6.0501 6.0501 0 0 0 -.91778.38383l-1.85881-1.23495-2.01924 2.01923 1.2388 1.86464a6.05267 6.05267 0 0 0 -.38067.91511l-2.18789.44119v2.85566l2.20054.44373a6.059 6.059 0 0 0 .37924.90383l-1.24251 1.87034 2.01923 2.01924 1.88089-1.24959a6.049 6.049 0 0 0 .8949.372l.44515 2.20735h2.85566l.44683-2.21567a6.05213 6.05213 0 0 0 .88907-.37186l1.882 1.25026 2.01923-2.01923-1.25089-1.88287a6.04854 6.04854 0 0 0 .37291-.89285zm-8.0053 1.61456a3.04782 3.04782 0 1 1 3.04782-3.04782 3.04781 3.04781 0 0 1 -3.04782 3.04782z"/>
+          </svg>
+        </button>
+        <article class="popup">
+          <fieldset>
+            <legend>Choose theme</legend>
+            <label tabindex="-1"><input type="radio" name="theme" value="light" /><span>Light</span></label>
+            <label tabindex="-1"><input type="radio" name="theme" value="dark" /><span>Dark</span></label>
+            <label tabindex="-1"><input type="radio" name="theme" value="light-hc" /><span>High contrast light</span></label>
+            <label tabindex="-1"><input type="radio" name="theme" value="dark-hc" /><span>Hight contrast dark</span></label>
+          </fieldset>
+        </article>
+      </div>
+    `;
```

Comment:
Check `Alert.html`. It had some overrides for theme-button:
![image](https://user-images.githubusercontent.com/36186912/157202537-72bf2491-8244-418c-8120-d75938edd9b3.png)
![image](https://user-images.githubusercontent.com/36186912/157202567-17a04687-d31d-42a9-938f-ed0b5c22e3d3.png)

Reply:
Should be fixed now